jekyll/_cci2/about-circleci.md

```diff
@@ -11,74 +10,83 @@ version:
+- Set up [parallelism]({{site.baseurl}}/2.0/parallelism-faster-jobs/) in your [.circleci/config.yml]({{site.baseurl}}/2.0/configuration-reference/) file to run jobs faster
+- Configure [caching]({{site.baseurl}}/2.0/caching/) with two simple keys to reuse data from previous jobs in your [workflow]({{site.baseurl}}/2.0/workflows/)
+- Configure self-hosted [runners]({{site.baseurl}}/2.0/runner-overview/) for unique platform support
+- Access [Arm resources]({{site.baseurl}}/2.0/arm-resources/) for the machine executor
+- Use [orbs]({{site.baseurl}}/2.0/using-orbs/), reusable packages of configuration, to integrate with third parties
+- Use pre-built Docker [images]({{site.baseurl}}/2.0/circleci-images/) in a variety of languages
+- Use the [API](https://circleci.com/docs/api/v2/) to retreive information about jobs and workflows
+- Use the [CLI]({{site.baseurl}}/2.0/local-cli/) to access advanced tools locally
+- Get flaky test detection with [test insights]({{site.baseurl}}/2.0/insights-tests/)
 
+As an operator or administrator of CircleCI installed on your own servers, CircleCI provides monitoring and insights into your builds and uses [Nomad](https://www.nomadproject.io/) for scheduling.
 
-## Free trial options
-{: #free-trial-options }
+See the [CircleCI Operations Guide]({{site.baseurl}}/2.0/server-3-overview/) for complete server documentation.
 
-CircleCI provides a free trial with the following options:
+## Pricing and trial options
+{: #pricing-and-trial-options }
 
-- **Cloud**: See [Signup and Try CircleCI]({{site.baseurl}}/2.0/first-steps/) to get started with the hosted application.
+Visit CircleCI's [Pricing page](https://circleci.com/pricing/) to view free and paid options. 
 
-### Open source
-{: #open-source }
+To get started with a free trial, visit the [Sign Up and Try CircleCI]({{site.baseurl}}/2.0/first-steps/) page and get set up with the hosted cloud application.
 
-See [Building Open Source Projects]({{site.baseurl}}/2.0/oss/) for information about free containers for public open source projects.
+Organizations on the free plan are given free credits to use on open source projects. Visit [Building Open Source Projects]({{site.baseurl}}/2.0/oss/) for more information about free containers for public open source projects.
 
-## See also
-{: #see-also }
+## Additional information
+{: #additional-information }
 
-Any app that runs on Linux, Android, or macOS is supported. Refer to the
-- [Supported Languages]({{site.baseurl}}/2.0/demo-apps/) document for examples and guides.
-- [Core Features]({{site.baseurl}}/2.0/features/) for detailed descriptions and links to procedural documentation.
+Any app that runs on Linux, Android, or macOS is supported. Further references:
+- [Supported Languages Guide]({{site.baseurl}}/2.0/demo-apps/) document for examples and guides
+- [Core Features]({{site.baseurl}}/2.0/features/) for detailed descriptions and links to procedural documentation
```

Comment:
OOo this is such a useful process - I think both this docs are very out of date and in need of reworking/archiving (I'll make notes of these). Instead let's link to: https://circleci.com/docs/2.0/tutorials/ and https://circleci.com/docs/2.0/concepts/ maybe?